src/v2/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -111,6 +125,40 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
     )
   }
 
+  renderZendeskScript() {
+    const {
+      is_acquireable,
+      is_in_auction,
+      isInquireable,
+      isOfferable,
+      listPrice,
+      priceCurrency,
+    } = this.props.artwork
+
+    const BNMO_CURRENCY_THRESHOLDS = {
+      USD: 10000,
+      EUR: 8000,
+      HKD: 77000,
+      GBP: 7000,
+    }
+
+    // This accounts for exact price and price ranges
+    const artworkPrice = listPrice?.major
+      ? listPrice.major
+      : listPrice?.minPrice?.major
+
+    if (is_in_auction) return
+    if (!is_acquireable && !isOfferable && !isInquireable) return
+    if (
+      (!artworkPrice && !isInquireable) ||
+      artworkPrice < BNMO_CURRENCY_THRESHOLDS[priceCurrency]
```

Comment:
This is interesting. For inquiry works we might not always have visible prices. Are we going to just enable chat for inquiry works when there is no visible prices? Or only enable chat for inquiry works that have visible prices _and_ the price is above the threshold?

Reply:
#non-blocking. Looks like similar logic is reused in the checkout flow and I wonder if there is an opportunity to DRY and move it to some kind of helper.

Reply:
> Or only enable chat for inquiry works that have visible prices and the price is above the threshold

This makes sense and maybe the statement should look like:

`!artworkPrice || artworkPrice < BNMO_CURRENCY_THRESHOLDS[priceCurrency]`

@starsirius what do you think?

Reply:
Yes that looks good to me! We can also be extra defensive and check if `priceCurrency` is actually defined in `BNMO_CURRENCY_THRESHOLDS`.

Reply:
```js
2 < null
// false
2 > null
// true ?!?!
2 < undefined
// false
2 > undefined
// false
```
😄 I think `artworkPrice < BNMO_CURRENCY_THRESHOLDS[priceCurrency]` will protect us from loading zendesk when currency is not defined but better to be explicit about it.